core/qbft/qbft.go

```diff
@@ -155,6 +155,7 @@ func Run[I any, V comparable](ctx context.Context, d Definition[I, V], t Transpo
 		dedupIn               = make(map[dedupKey]bool)
 		timerChan             <-chan time.Time
 		stopTimer             func()
+		dedupRules            = make(map[uponRule]bool)
```

Comment:
nit: I would put the two dedup maps next to each other. We could also rename `dedupIn` to `dedupMsgs`

Reply:
Done